docs/monitoring/MonitoringMigrationDashboard.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Migration Monitoring Dashboard
+parent: Monitoring
+nav_order: 2
+---
+# Migration Monitoring Dashboard
+
+The Migration Monitoring Dashboard is a custom dashboard created to provide greater visibility into the health and progress of various components of the migration.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Components in Monitoring Dashboard
+
+<img src="https://services.google.com/fh/files/misc/mon-dashboard-main.png"  style="width:800px;"/>
+
+### Overview
+The first section of the monitoring dashboard provides key graphs for insights on the migration progress.
+
+|Resource | Metric | Description | Aggregation | Relevance |
+|-----------|-----------|-----------|-----------|-----------|
+| Dataflow | Worker CPU Utilization | Shows the CPU Utilization of a dataflow worker| - 50th percentile worker CPU Utilization<br> - 90th percentile worker CPU Utilization<br> - Max percentile worker CPU Utilization<br> | Used to identify if the pipelines is over or under scaled based on the value of CPU Utilization| 
+| Datastream | Throughput(events/sec) | Shows the average events processed/sec generated at source | Average | Used to track if data is being transferred from source to GCS Bucket| 
+| Datastream | Unsupported Events | Source events unsupported by Datastream | Sum | Used to identify if there is any data that can't be transferred by datastream due to a correctness issue|
+| Pubsub | Age of Oldest Unacknowledged Message | Age of the oldest unacknowledged message in the subscription | Max | Used to determine if starvation of dataflow resources is taking place |
+| Spanner | CPU Utilization | CPU Utilization of spanner database and instance| - Database Total CPU Utilization<br> - Instance Total CPU Utilization<br> | Used to track if spanner is overloaded and requires more or less nodes | 
+| Spanner | Storage | Storage of spanner database and instance | - Database Total Storage<br> - Instance Total Storage<br> | Used to track how the data is growing as the migration proceeds |
+
+### Dataflow Metrics
+
```

Comment:
Do you think each section should contain the respective screenshot for it? 
The current screenshot on this page has only the overview page expanded.